### PR TITLE
[pvr] bumps addons to use libplatform

### PR DESCRIPTION
**Pull request by hudokkow**

### CODE REVIEW

**Comment by hudokkow** [comment]
jenkins build addons please

File: project/cmake/addons/addons/pvr.vdr.vnsi/pvr.vdr.vnsi.txt [hunk 1]
```diff
@@ -1 +1 @@
-pvr.vdr.vnsi https://github.com/kodi-pvr/pvr.vdr.vnsi 3a28e39
+pvr.vdr.vnsi https://github.com/kodi-pvr/pvr.vdr.vnsi 4c6ef26
```
Comment (FernetMenta) [inline project/cmake/addons/addons/pvr.vdr.vnsi/pvr.vdr.vnsi.txt]: new sha 153fb125610034361248d08ed18dab603f7a4c42

**Comment by Memphiz** [comment]
isn't #7069 needed to make this work with the addons build trigger? (without it the libplatform is not built at all...)

**Comment by ksooo** [comment]
@memphiz I think you are right.

**Comment by ksooo** [comment]
#7069 must go in first, then the bumped add-ons can be built again (by Jenkins).

**Comment by ksooo** [comment]
wtf?

**Comment by ksooo** [comment]
My phone is acting up badly.... Sorry.

**Comment by opdenkamp** [comment]
indeed, they both have to go in at the same time. if you want to test a build, cherry-pick https://github.com/xbmc/xbmc/pull/7069

**Comment by hudokkow** [comment]
Superseded by https://github.com/xbmc/xbmc/pull/7069

**Pull request closed without merging**